parameterized/parameterized.py

```diff
@@ -19,9 +19,14 @@
     class SkipTest(Exception):
         pass
 
+try:
+    import pytest
+except ImportError:
+    pytest = None
+
 PY3 = sys.version_info[0] == 3
 PY2 = sys.version_info[0] == 2
-
+PYTEST4 = pytest and pytest.__version__ >= '4.0.0'
```

Comment:
I would also note that e.g. '10.0.0' is lower than '4.0.0'. Something like this should work:

```suggestion
PYTEST4 = pytest and int(pytest.__version__.split('.')[0]) >= 4
```